lib/grizzly/zwave/commands/user_code_report.ex

```diff
@@ -50,7 +50,7 @@ defmodule Grizzly.ZWave.Commands.UserCodeReport do
      [
        user_id: user_id,
        user_id_status: UserCode.user_id_status_from_byte(user_id_status_byte),
-       user_code: user_code_binary
+       user_code: String.trim(user_code_binary, <<0>>)
```

Comment:
```suggestion
       user_code: String.trim_trailing(user_code_binary, <<0>>)
```
[`String.trim/2` is both leading and trailing](https://hexdocs.pm/elixir/String.html#trim/1) but the description says trailing. Did you really want both?

Reply:
Good catch.